### ISSUE TITLE
How to process ground-truth depth when training?

### ISSUE THREAD

**Issue by luohongcheng** [iro-cp/FCRN-DepthPrediction#49]
Hi~ 
_the original frames of size 640 × 480 pixels are down-sampled to 1/2 resolution and center-cropped to 304 × 228 pixels_
When I do the same operation to ground-truth, It seems that the black border still exists. 
How to process ground-truth depth？

**Comment by irolaina** [comment]
Indeed, there exists an invalid depth border for the ground truth. We mask it out during training, along with all the other invalid pixels. We have not filled in any pixel values

**Comment by ieted** [comment]
I notice that you rescale your output depth prediction from 160x128 to 640x480 to compute the loss (against the untouched ground truth depth map), however, the predicted depth is in correspondence with the cropped version of the input RGB image, wouldn't there be some pixel location bias? @iro-cp

**Comment by irolaina** [comment]
Hi, actually we did not crop the input RGB and thus rescaling the prediction to full resolution should be in alignment with the ground truth. Differently to other papers in depth estimation, we did not compute errors in a center-cropped region only, but in the full image. For completeness we will report those errors too.

**Comment by ieted** [comment]
In the first paragraph of the 4.2 subsection of your paper, there is a sentence 
> ...Following [6], the original frames of size 640x480 pixels are down-sampled to 1/2 resolution and center-croped to 304x228 pixels, as input to the network.

that states you center-cropped the input RGB image in the experiment. If you did not crop the image, then what the operation should be after you down-sample the 640x480 image to 320x240 to get a 304x228 image as the input of the network?
